scriptoocloud/src/main/java/edu/asu/diging/scriptoocloud/config/SecurityContext.java

```diff
@@ -37,7 +37,8 @@ protected void configure(HttpSecurity http) throws Exception {
                 .antMatchers("/", "/resources/**",
                         "/register").permitAll()
                 // The rest of the our application is protected.
-                .antMatchers("/users/**", "/admin/**").hasRole("ADMIN")
+                .antMatchers("/users/**", "/admin/**","/auth/**").hasRole("ADMIN")
+                .antMatchers("/auth/add","/auth/remove").hasRole("USER")
```

Comment:
You need to move `/auth/**` to its own antMatcher and then use `hasAnyRole("USER", "ADMIN")`